src/components/ae-slide/ae-slide.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <agile class="ae-slide" :class="{ fullscreen }" :options="defaults">
+    <slot />
+  </agile>
+</template>
+<script>
+import { Agile } from 'vue-agile';
+
+/**
+ * The ae-slide uses vue-agile under the hood: https://github.com/lukaszflorczak/vue-agile
+ *
+ * The option is the configuration object, where you can modify
+ * the behavior of the vue-agile.
+ *
+ * Currently this is a simple wrapper on top of the vue-agile
+ * with some stylistic changes, and acts like a interface
+ * in case in the future we will have to change / implement our
+ * own slider.
+ */
+export default {
+  name: 'ae-slide',
+  components: { Agile },
+  props: {
+    /**
+     * You can modify the behavior of the slider
+     * using the following options:
+     * @link https://github.com/lukaszflorczak/vue-agile#options
+     */
+    options: Object,
+
+    /**
+     * Creates a full screen slider, where the element tries to go
+     * 100% height and 100% width of the parent element, and places
+     * the round dots inside of the slider
+     */
+    fullscreen: Boolean,
+  },
+  computed: {
+    /**
+     * Returns the default object
+     * configuration for vue-agile
+     * @return {{}}
+     */
+    defaults() {
+      return Object.assign({
+        arrows: false,
+        infinite: false,
+      }, this.options);
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+  @import '../../styles/globals';
```

Comment:
please remove extra 2 space indents in `<styles>` block in consistency with `<script>`